### PR TITLE
Added Initial ScholarX Mentor List - 2021

### LINKED ISSUE #833: Add the first set of Mentors 2021
** Please describe.**
The first set of mentors are confirmed. So this request is to add the first set of Mentors to the ScholarX 2021 page

**Are you working on this?**
Yes, I am

### PR DESCRIPTION
**Pull request by NaviRocker**
## Purpose

The purpose of this PR is to fix #833 

## Goals

Added the Initial Approved Mentor List

### Preview Link

https://pr-834-sef-site.surge.sh/

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines ( http://bit.ly/sef-best-practices )
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
 

## Test environment
Google Chrome in Windows 8.1

Please squash and merge the PR

### CODE REVIEW

File: src/main/webapp/scholarx/2021/mentors.json [hunk 1]
```diff
@@ -364,4 +364,4 @@
       }
     ]
   }
-]
+]
```
Comment (janethavi) [inline src/main/webapp/scholarx/2021/mentors.json]: Can you please add the EOF line here?

**Review by janethavi — CHANGES_REQUESTED** [review]

**Comment by janethavi** [comment]
There are some thumbnail files. Do we need them to work properly?
eg:-  `src/main/webapp/scholarx/2021/images/Thumbs.db `

**Review by Gimhan-minion — COMMENTED** [review]
@NaviRocker I guess we can remove those Thumbs files.

**Comment by NaviRocker** [comment]
Thumbs. db is an image cache that makes thumbnail viewing faster. So I think it might be a bit useful for this.
@janethavi  @Gimhan-minion

**Pull request closed without merging**